addons/mail/static/src/new/thread/thread.xml

```diff
@@ -10,8 +10,8 @@
                 <t t-if="thread.loadMore and props.order === 'asc'" t-call="mail.thread.load_more"/>
                 <t t-foreach="props.order === 'asc' ? thread.messages : [...thread.messages].reverse()" t-as="messageId" t-key="messageId">
                     <t t-set="msg" t-value="messaging.state.messages[messageId]"/>
-                    <t t-if="msg.dateDay !== currentDay">
-                        <div class="d-flex fw-bolder pt-4">
+                    <t t-if="!msg.isEmpty and msg.dateDay !== currentDay">
+                        <div class="o-channel-day-separator d-flex fw-bolder pt-4">
```

Comment:
I believe it's not specific to channel (in regard to class name).

Reply:
change done